README.md

```diff
@@ -19,28 +19,34 @@ For issues see [Snovault OSX App Installation][] first.
     $ source encoded-venv/bin/activate
     ```
 
-1. Clone the repo and install requirements
+2. Clone the repo and `cd` into it
     ```
-    # Make sure you are in the encoded-venv
+    $ git clone git@github.com:ENCODE-DCC/encoded.git
     $ cd encoded
-    $ pip install -r requirements.osx.txt
     ```
 
-1. Build Application
+3. Build Application
     ```
-    $ make clean && buildout bootstrap && bin/buildout
+    $ make clean && pip install -e '.[dev]' && buildout
     ```
 
-1. Run Application
+If you need to develop `snovault` side by side you can use the following commands instead, assuming `encoded` and `snovault` are present at the same level in your filesystem and virtual environment is activated.
```

Comment:
Does "at the same level" mean that I have  repos located like `/home/user/repos/encoded/` and `/home/user/repos/snovault/`?

Reply:
yes